test/grid_space_tests.jl

```diff
@@ -63,6 +63,11 @@ using StableRNGs
         random_positions = positions(model, :random)
         @test all(n ∈ pos_map for n in random_positions)
```

Comment:
```suggestion
        @test all(n in pos_map for n in random_positions)
```
For consistency and simplicity, prefer `in` to `∈`.